### PR TITLE
Rcswitch

### PR DESCRIPTION
**Pull request by bmorcelli**
#### Proposed Changes ####

Read and save RAW data

#### Types of Changes ####

structural

#### Linked Issues ####

Many

#### User-Facing Change ####

Users can now save RAW signals and replay after read at any time.

### CODE REVIEW

**Comment by Petitoursbr1** [comment]
Yessss thank you so much

File: platformio.ini [hunk 1]
```diff
@@ -66,7 +66,7 @@ lib_deps =
 	Timezone
 	ESP32Time
 	bblanchon/ArduinoJson
-	rc-switch
+	https://github.com/bmorcelli/rc-switch
```
Comment (rennancockles) [inline platformio.ini:69]: @bmorcelli reescrevendo todas as libs pro Bruce haha 🔥🔥🔥

File: src/core/settings.cpp [hunk 2]
```diff
@@ -63,6 +63,7 @@ int gsetRotation(bool set){
   if(set) {
     bruceConfig.setRotation(result);
     tft.setRotation(result);
+    tft.setRotation(result);
```
Comment (rennancockles) [inline src/core/settings.cpp:66]: resolveu??

File: platformio.ini (hunk 1 above)
Comment (bmorcelli) [reply platformio.ini:69]: Pau na máquina!!

**Pull request merged**